packages/ts/react-grid/src/autocrud.tsx

```diff
@@ -0,0 +1,58 @@
+import type { AbstractModel, DetachedModelConstructor } from '@hilla/form';
+import type { GridElement } from '@hilla/react-components/Grid.js';
+import { HorizontalLayout } from '@hilla/react-components/HorizontalLayout.js';
+import { createContext, useState, type JSX } from 'react';
+import { ExperimentalAutoForm } from './autoform';
+import { AutoGrid } from './autogrid';
+import type { CrudService } from './crud';
+
+export type AutoCrudProps<TItem> = Readonly<{
+  service: CrudService<TItem>;
+  model: DetachedModelConstructor<AbstractModel<TItem>>;
+}>;
+
+interface AutoCrudContextType<TItem> {
+  item: TItem | undefined;
+}
+
+export const AutoCrudContext = createContext<AutoCrudContextType<any>>({ item: undefined });
+
+export function ExperimentalAutoCrud<TItem>({ service, model }: AutoCrudProps<TItem>): JSX.Element {
+  const [item, setItem] = useState<TItem | undefined>(undefined);
```

Comment:
What do you want to distinguish it from? To me it is the `item` that is being used by the CRUD. As grid uses both `selectedItem` and `activeItem` we should not confuse it with those. `currentItem` could work but that sounds a bit like there would be a `previousItem` also? It could be `currentItem`

Reply:
As `AutoForm` also uses `item`, I would not rename it as part of this PR at least

Reply:
The way I was picturing the AutoForm or a form in general, is a navigable form with <kbd> << </kbd> <kbd> < </kbd> [ x of n record(s) ] <kbd> > </kbd><kbd> >> </kbd> (pagination controls) so that I can move back and forward without relying on grid. 
Let's think about having the `currentItem` in that context, but not rename it here.